dotCMS/src/main/java/com/dotcms/ema/proxy/MockHttpCaptureResponse.java

```diff
@@ -0,0 +1,57 @@
+package com.dotcms.ema.proxy;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+import javax.servlet.ServletOutputStream;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpServletResponseWrapper;
+
+import com.dotcms.mock.response.MockResponse;
+
+
+/**
+ * Proxy for HTTPServletResponse
+ *
+ */
+public class MockHttpCaptureResponse extends HttpServletResponseWrapper implements MockResponse {
+
+
+    ByteArrayOutputStream bout = new ByteArrayOutputStream(4096);
+    ServletOutputStream out = null;
```

Comment:
Codacy found an issue: [Avoid using redundant field initializer for 'out'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5922984)